src/main/java/org/sefglobal/scholarx/model/Comment.java

```diff
@@ -0,0 +1,58 @@
+package org.sefglobal.scholarx.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.GenerationType;
+import javax.persistence.ManyToOne;
+
+@Entity
+@Table(name = "comment")
+public class Comment {
```

Comment:
We can extend the `BaseScholarxModel` and remove the id from here

Reply:
oh yeah